src/lib/validation/recipient.ts

```
@@ -40,18 +36,14 @@
 
         case 'USERNAME':
             const isValidPeanutUsername = await verifyPeanutUsername(recipient)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)